doc/getting-started/apl-nuc.rst

```diff
@@ -177,7 +177,8 @@ partition. Follow these steps:
       whether it should be disabled. There are three forms for this parameter:
 
       #. ``uart=disabled``: this disables the serial port completely
-      #. ``uart=mmio@<MMIO address>``: this sets the serial port MMIO address
+      #. ``uart=bdf@<BDF value>``:  this sets the PCI serial port based on its BDF.
+           For example, use ``bdf@0:18.2`` for a BDF of 0:18.2 ttyS2.
```

Comment:
Just a couple too many spaces in the indentation, the beginning of the line should align with the first tick character